Fixed #12437 -- Added css_classes to Form._html_output().

alimony opened the pull request:
See ticket: https://code.djangoproject.com/ticket/12437

MoritzS commented on django/forms/forms.py:
```diff
@@ -249,7 +250,8 @@ def _html_output(self, normal_row, error_row, row_ender, help_text_html, errors_
                         'label': '',
                         'field': '',
                         'help_text': '',
-                        'html_class_attr': html_class_attr,
```
Why did you make the `html_class_attr` field empty here?

alimony replied:
I saw no real reason for it when creating an empty row for the sole purpose to insert other output. However, I digged through the tickets I could find on this (3512, 3686, 3515), and looked at the [original commit](https://github.com/django/django/commit/92803205cbcaaee16ac0eb724c45019a9d896aac) adding it, and it seems like it was written like that on purpose, although I find no reference to why. So I just pushed a new commit that reverts it, doesn't make a difference for this new patch :)

timgraham commented:
buildbot, add to whitelist.

timgraham commented on tests/forms_tests/tests/test_forms.py:
```diff
@@ -2274,11 +2274,50 @@ class SomeForm(Form):
+
+    def test_field_with_css_class(self):
+        class SomeForm(Form):
+            some_field = CharField(widget=TextInput(attrs={'class': 'first_name_id'}))
```
is `widget=...` important for the test?

alimony commented:
I think the widget was just a leftover from my dev process, I've removed it now and squashed some of the commits. Should I ideally squash everything into one commit?

timgraham commented:
Yes, please. You can also use the commit message from the title of the pull request (see the [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist) for the commit message guidelines).

alimony commented:
I've squashed everything into one commit with a better message, will pay more attention to this in the future.

timgraham commented:
merged in 1884bf8e8e9fb3f9f44a9ec216aca96f63974268, thanks!
